app/i18n/community_translators.md

```diff
@@ -0,0 +1,125 @@
+# Communitity Translators
+
+Decrediton translations are handled by the various localized communities. Please note that translations are only accepted from **well-known** community members.
+
+This document describes the process such community members may take to perform a Decrediton translation.
+
+Two translation flows are documented here:
+
+- **Basic**: Suitable for smaller communities, that don't have a technical member that can keep up to date with the building requirements of Decrediton.
+- **Advanced**: Preferable when the community member is a software developer or can be relied to be self-sufficient in learning the tooling around building and running a development version of Decred software.
+
+The main limitation of the basic UI translation flow is that it doesn't support testing the translation of the more extensive docs (such as tutorials, info modals, etc) and it depends upon the release of a Release Candidate (RC) version of Decrediton to be provided by the development team. This makes it more likely that the translation will end up missing some final strings included in changes made very close to the final released version and the translators might have to write the translations on a tight schedule.
+
+The advanced translation flow requires setting up a local building infrastructure for Decrediton. This setup is more involved, recommended when the translator is a software developer (and thus already have experience doing these kinds of setups) or is a self-sufficient technical person and can figure most things by themselves. The main advantage is that this method allows doing translations at any time during development and to immediately test any necessary adjustments to the translation.
+
+The next sections detail both translation flows.
+
+## Basic UI Translation
+
+Basic UI translation can be done once an RC (release candidate) version of Decrediton is released.
+
+Translators have to fulfill the following requirements:
+
+- Create a [Github](https://github.com) account
+- Install [git software](https://guides.github.com/introduction/git-handbook/)
+- Download the latest released Decrediton RC version
+- Follow the [#decrediton](https://matrix.to/#/#decrediton:decred.org) matrix chat room for announcements
```

Comment:
Optionally, also mention `#translations` chat room.

Reply:
Changed to `#translations` exclusively. I believe that's the appropriate channel for translators to decred (`#decrediton` might be too noisy).